### PR TITLE
Bring demographics schema in line with data model standard spec

### PR DESCRIPTION
**Pull request by khmoran**
- Species has been nixed
- Profession, nationality, and ethnicity have been added: All optional string fields. In the future should be validated against a data dictionary.

In the future, profession and nationality will be validated against data dictionaries (no free text entry); ethnicity will have a data dictionary but the option to write in free text

### CODE REVIEW

**Comment by khmoran** [comment]
[Updates to schema based on standard spec](https://trello.com/c/uimr21R4/207-updates-to-schema-based-on-standard-spec)

File: data-serving/data-service/src/model/demographics.ts [hunk 1]
```diff
@@ -24,18 +20,20 @@ export const demographicsSchema = new mongoose.Schema({
...
     },
-    species: {
-        type: String,
-        enum: Object.values(Species),
-    },
     sex: {
         type: String,
         enum: Object.values(Sex),
     },
+    // TODO: The below 3 fields should be data dictionaries.
```
Comment (axmb) [inline data-serving/data-service/src/model/demographics.ts:27]: Even profession? Seems tough.

File: data-serving/data-service/test/model/data/case.full.json [hunk 2]
```diff
@@ -5,7 +5,9 @@
             "end": 59.0
         },
         "sex": "Female",
-        "species": "Homo sapien"
+        "profession": "Horse trainer",
```
Comment (axmb) [inline data-serving/data-service/test/model/data/case.full.json:8]: My point exactly (re: dictionary). :-P

File: data-serving/data-service/src/model/demographics.ts (hunk 1 above)
Comment (khmoran) [reply data-serving/data-service/src/model/demographics.ts:27]: Can ask Stephen but currently there's a whole list here: https://docs.google.com/spreadsheets/d/1qOnGJQg1gtbmNxAW8c9hC1TvFpIQmSnORD-x-iXrpJ0/edit?ts=5ee23b03#gid=1189434977

File: data-serving/data-service/test/model/data/case.full.json (hunk 2 above)
Comment (khmoran) [reply data-serving/data-service/test/model/data/case.full.json:8]: Could've gone with "Jockey – steeple", just saying.

**Pull request merged**